src/components/Button/Button.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon } from '../Icon/Icon';
+
+import './Button.scss';
+
+export const Button = ({
+  toggle,
```

Comment:
It would be better to rename this prop to something more reusable, since the buttons in this app won't always toggle things